tests/acceptance/test_build.py

```diff
@@ -1102,3 +1102,11 @@ def test_build_nodbus(self, prepared_test_build, bitbake_path):
 
         # Actual test.
         assert "libglib" not in output
+
+        # Make sure busconfig files are also gone.
```

Comment:
Same as https://github.com/mendersoftware/mender-image-tests/pull/20#discussion_r520401810, we need to bump `min_mender_version` in the test

Reply:
Actually, this one we should keep, because it tests for the *absence* of the files, which is correct also for older versions. It makes sense to test those versions as well to make sure the new DBus dependency code we added to `mender-client.inc` (which is shared) has not leaked into the old versions.

Reply:
You are completely right.